File: sistema-fenix-front-end/src/app/shared/components/model/ProductModel.ts
export class ProductModel {
  productId: number;
  productName: string;
  status: boolean;
  quantity: number;
  movementType: string;

  constructor() {
    this.productId = 0;
    this.productName = '';
    this.status = false;
    this.quantity = 0;
    this.movementType = '';
  }
}

File: sistema-fenix-back-end/src/main/java/com/spring/security/jwt/model/ProductModel.java
package com.spring.security.jwt.model;

public class ProductModel {
    private Long productId;
    private String productName;
    private boolean Status;
    private Integer quantity;
    private String movementType;
    
	public Long getProductId() {
		return productId;
	}
	public void setProductId(Long productId) {
		this.productId = productId;
	}

	public String getProductName() {
		return productName;
	}
	public void setProductName(String productName) {
		this.productName = productName;
	}
	public boolean getStatus() {
		return Status;
	}
	public void setStatus(boolean status) {
		Status = status;
	}
	public Integer getQuantity() {
		return quantity;
	}
	public void setQuantity(Integer quantity) {
		this.quantity = quantity;
	}
	public String getMovementType() {
		return movementType;
	}
	public void setMovementType(String movementType) {
		this.movementType = movementType;
	}

    
    
}
